Shared/Samples/Show geodesic sector and ellipse/ShowGeodesicSectorAndEllipseView.swift

```diff
@@ -0,0 +1,338 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowGeodesicSectorAndEllipseView: View {
+    /// The data model that helps determine the view.
+    @StateObject private var model = Model()
+    
+    /// Manages the presentation state of the menu.
+    @State private var isPresented: Bool = false
+    
+    var body: some View {
+        MapViewReader { proxy in
+            MapView(
+                map: model.map,
+                graphicsOverlays: model.graphicOverlays
+            )
+            .onSingleTapGesture { _, tapPoint in
+                model.centerPoint = tapPoint
+            }
+            .task(id: model.centerPoint) {
+                guard let centerPoint = model.centerPoint else { return }
+                await proxy.setViewpoint(
+                    Viewpoint(center: centerPoint, scale: 1e7)
+                )
+            }
+            .toolbar {
+                // The menu which holds the options that change the ellipse and sector.
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Button("Settings") {
+                        isPresented = true
+                    }
+                    .disabled(model.centerPoint == nil)
+                    .popover(isPresented: $isPresented) {
+                        Form {
+                            ParameterSlider(
+                                label: "Axis Direction:",
+                                value: $model.axisDirection,
+                                range: 0...360
+                            )
+                            .onChange(of: model.axisDirection) {
+                                model.refreshSector()
+                            }
+                            
+                            Stepper(
+                                value: $model.maxPointCount,
+                                in: 0...1000,
+                                step: 1
+                            ) {
+                                Text("Max Point Count:  \(String(format: "%d", model.maxPointCount))")
```

Comment:
If you use a slider this wouldn't matter, but this text initializer takes a value of `LocalizedStringKey`, which has its own string interpolation type with an interpolation [method](https://developer.apple.com/documentation/swiftui/localizedstringkey/stringinterpolation/appendinterpolation(_:format:)) that takes a format style. In other words, the way to do this without using `String.init(format:_:)` is: `Text("Max Point Count: \(model.maxPointCount, format: .number)")`.

Reply:
I think if the all the settings being visible at the same time is no longer on the agenda, then it makes sense to completely change the UI elements.

Reply:
Udated, now uses slider component.